canonical-websites/snapcraft.io
# Release UI - expandable history panel

Fixes #1267 

Opens releases history in expaning panel inside releases table and updates the styles.

#### DONE:

- [x] open history in expanding panel
- [x] update styles for 'tabs'
- [x] rename things (get rid of overlay related stuff)
- [x] open/close on cell click
- [x] clean up naming of css classes
- [x] remove arch column (when arch is filtered)
- [x] heading with `arch – channel`
- [x] hide indent of panel on smaller screens
- [x] don't indent first column without checkboxes

### QA

- ./run or https://snapcraft-io-canonical-websites-pr-1315.run.demo.haus/
- open Releases page of any snap
- click on any channel/arch cell to see history of releases
- click on 'unassigned revisions' to see all available revisions for given arch

<img width="1025" alt="screen shot 2018-11-15 at 17 00 40" src="https://user-images.githubusercontent.com/83575/48565113-0c423880-e8f8-11e8-8918-c47430592f9b.png">
